src/components/CR8 - GalleryText/GalleryText.vue

```diff
@@ -0,0 +1,247 @@
+<template>
+    <div class="gallery-text">
+        <!-- Title -->
+        <h2 class="title">Our Ecosystem</h2>
+
+        <!-- Subtitle w/ Arrow -->
+        <div class="subtitle">
+            <span class="subtitle-text">We develop and produce original content for all genres and formats.</span>
+            <span class="prompt">
+                <span class="text" >explore</span>
+                <svg-arrow-right class="arrow" />
+            </span>
+        </div>
+        
+        <!-- Block Links -->
+        <div class="text-block">
+            <span
+                v-for="company in companies"
+                :key="company.name"
+            >
+                <nuxt-link class="company-link" :to="`/${company.name}`" tag="a">
+                    <component 
+                        :is="`logo-${company.name}`"
+                        class="company-logo"
+                        @mouseover="set(company)" 
+                        @mouseout="set(null)" />
+                </nuxt-link>
+                <span class="text">{{company.text}} </span>
```

Comment:
Does this need to be in a `span`, can it just be `{{company.text}}`?

Reply:
I am wrapping this with a span with a class .text so that I can target it on the company logo hover, the text needs to have a higher z-index when the company logo is hovered over.  in SCSS, using ~ .text to target the sibling child on company-logo hover.